### ISSUE TITLE
Pending requests with azure WAF

### ISSUE THREAD

**Issue by antoinechamot** [axios/axios#6459]
### Describe the issue

I'm encountering an issue where I make a POST request using axios, but it gets blocked by Azure WAF, resulting in a 403 status code. Following this, any subsequent requests I attempt to make with axios do not proceed and are indefinitely stuck in a pending state.
It appears as though the connection is perpetually frozen on the request that Azure rejected.
What could be causing this, and how can I avoid this situation?

**Comment by igstan** [comment]
This issue might be addressed by #6661, at least the first part of it where requests are being blocked by Azure WAF.

**Comment by jasonsaayman** [comment]
this has been merges so it should be fixed, if not please open a new issue and mention me
